prep_inputs_for_combiner.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python
+
+"""
+Uploads GVCFs and QC files to CPG and senerates sample maps for CPG-QC
```

Comment:
```suggestion
Uploads GVCFs and QC files to CPG and generates sample maps for CPG-QC
```
Also: is that CPG-QC meant to be joint-calling?